docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -241,41 +241,62 @@ Step 2: Send **at least 1 Holesky ETH** to the “address” field referenced in
 
 #### Configuration Setup
 
-You can create the config files needed for operator registration using the below command:
+**Step 1:** Create the config files needed for operator registration using the below command:
 
 ```
 eigenlayer operator config create
 ```
 
 When prompted for operator address, make sure your operator address is same as the ecdsa key address you created/imported in key creation steps. 
 
-It will create two files: `operator.yaml` and `metadata.json` After filling the details in `metadata.json`, please upload this into a publicly accessible location and fill that url in `operator.yaml`. A valid metadata url is required for successful registration. An example operator.yaml file is provided for your reference here: [operator.yaml](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/operator-config-example.yaml) .
+The command will create two files: `operator.yaml` and `metadata.json`.
 
-A public metadata url is required to register the operator. After updating your `metadata.json` file, you must upload it to a publicly accessible location and add that public url to your `operator.yaml` file. You are also required to upload the image of the operator to a publicly accessible location and give the url in the metadata file. Operator registration only supports `.png` images for now and must be less than 2MB in size.
+**Step 2:** Modify `metadata.json`, upload, and ensure it is publicly accessible:
+
+After filling the details in `metadata.json`, please upload this into a publicly accessible location and fill that url in `operator.yaml`. Please note that a **publicly accessible** metadata url is required for successful registration. An example operator.yaml file is provided for your reference here: [operator.yaml](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/operator-config-example.yaml) .
+
+**Step 3:** Upload operator logo and ensure it is publicly accessible:
+
+You are also required to upload the logo of the operator to a publicly accessible location and set the url in your `metadata.json` file. The `metadata.json` must be less than 4KB in size. Operator registration only supports `.png` images for now and must be less than 1MB in size.
+
```

Comment:
should this info be before we ask them to publicly upload in Step 2?

Reply:
Good point. Reorganized accordingly